test/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/metal3-io/baremetal-operator/test
 go 1.20
 
 require (
+	github.com/cert-manager/cert-manager v1.10.0
```

Comment:
The version seems strange considering that we have v1.13 in the e2e config.

Reply:
Yes. Basically using newer versions would trigger upgrades in many of the dependencies, which'd make everything a mess :-s 1.10 is the newest one I could find that didn't trigger too many breaks.

Reply:
My apologies. Apparently I tried many other versions but not 1.13 :sweat_smile: It works.

Reply:
Ok I see. We can take this for now probably, but at some point we will need to bump it